### ISSUE TITLE
Accuracy of MobileNet_v2 evaluation

### ISSUE THREAD

**Issue by Sangsooko** [tensorflow/models#4809]
I evaluated Slim models in my GPU server with pre-trained checkpoint in TF slim models.
But, Top-1 Accuracy of MobileNet_v2_1.0_224 is too lower than that in README.
The gap is 1.774% ( Top-1 Accuracy in README = 71.9, The result in my GPU server = 70.13 )

Please let me know what I should do to get same top-1 accuracy of MobileNet_v2.

Please check the following table to check my results.
-------------------------+----------+-----------+---------
,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,|_Top-1 Accuracy(%)__| 
-------------------------+----------+-----------+---------
model,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,|_readme_ |_by myself_|_difference 
-------------------------+----------+-----------+---------
Inception_V3,,,,,,,,,,,,,,,,,,,,,|____78.00 |______77.98 |____0.022
VGG16,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,|____71.50 |______70.80 |____0.700
VGG19,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,|____71.10 |______71.00 |____0.100
NASNet-A_Mobile_224,|____74.00 |______73.97 |____0.030
MobileNet_v1_1.0_224,,|____70.90 |______70.75 |____0.152
MobileNet_v2_1.4_224,,|____74.90 |______74.11 |____0.790
MobileNet_v2_1.0_224,|____71.90 |______70.13 |____1.774
-------------------------+----------+-----------+---------

readme data of Top-1 Accuracy is from the following link.
https://github.com/tensorflow/models/tree/master/research/slim#Pretrained

other settings are following.
TF version 1.8, ILSVRC-2012, latest tensorflow/models codes and check point.

The following is setting for eval_image_classifier.py
// ==================================
python eval_image_classifier.py \
--checkpoint_path=${CHECKPOINT_DIR} \
--eval_dir=${EVAL_DIR} \
--dataset_dir=${DATASET_DIR} \
--dataset_name=imagenet \
--dataset_split_name=validation \
--preprocessing_name=mobilenet_v1 \
--model_name=mobilenet_v2 \
--eval_image_size=224

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by Sangsooko** [comment]
Thank you for your response. please check the followings. if you need more information, please let me know that.

What is the top-level directory of the model you are using --> ~/tensorflow/models/research/slim
Have I written custom code ----> No Custom code
OS Platform and Distribution --> Linux ubuntu 4.4.0-87-generic 
TensorFlow installed from -----> virtualenv + pip install (tensorflow-gpu)
TensorFlow version ------------> 1.8.0
Bazel version -------------------> Build label: 0.12.0
CUDA/cuDNN version ---------> CUDA=V9.0.176,  CuDNN = 7.0.4
GPU model and memory ------> Tatan Xp, 12GByte
Exact command to reproduce --> 
export DATASET_DIR= my tfrecode dir of  ILSVRC-2012
export EVAL_DIR=~/tensorflow/models/research/slim/eval
export CHECKPOINT_DIR=(uncompressed folder of mobilenet_v2_1.0_224.tgz in TF model GITHUB)/mobilenet_v2_1.0_224.ckpt  # Example
python eval_image_classifier.py \
--checkpoint_path=${CHECKPOINT_DIR} \
--eval_dir=${EVAL_DIR} \
--dataset_dir=${DATASET_DIR} \
--dataset_name=imagenet \
--dataset_split_name=validation \
--preprocessing_name=mobilenet_v1 \
--model_name=mobilenet_v2 \
--eval_image_size=224

**Comment by qingyangqing** [comment]
I have the same problem. My top-1 accuracy of MobileNet_v2_1.0_224 is 70.126%.

**Comment by netanel-s** [comment]
@bitfort , 5 months have passed since the issue was opened, any update?
Thanks in advance.

**Comment by wenmengzhou** [comment]
+1  my mobilenet_v2 top-1 accuracy  0.701   top5-accuracy 0.895

**Comment by cococyx** [comment]
If I use imagenet_accuracy_eval, Top-1 Accuracy of mobilenet_v1_1.0_224.tflite is 0.70950. But Top-1 Accuracy of MobileNet_v1_1.0_224_quant is 0.6805. The gap is  2.85%.
So The Accuracy on web is based on *.ckpt or *.pb or *.tflite ?

**Comment by ExirLiu** [comment]
Similarly, I achieved 74.11% top-1 accuracy on mobilenetv2 1.4. 
I think uploading an unified validation set in tfrecode format (about 7Gb) would be helpful,
Cause I download the imageNet dataset from kaggle then convert it into tfrecode by part of the official codes.

**Comment by ExirLiu** [comment]
By restoring ExponentialMovingAverage of each parameter when evaluation can achieve the acc in README!

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
